packages/behandling-pleiepenger/src/components/MedisinskVilkår.tsx

```diff
@@ -24,12 +27,14 @@ const initializeMedisinskVilkår = (
     onFinished: løsAksjonspunkt,
     readOnly,
     visFortsettknapp,
+    saksbehandlere,
   });
 };
 
 const medisinskVilkårAppID = 'medisinskVilkårApp';
 export default ({ behandling: { links, uuid }, submitCallback, aksjonspunkter, readOnly }) => {
   const { addErrorMessage } = useRestApiErrorDispatcher();
+  const saksbehandlere = useGlobalStateRestApiData<any>(K9sakApiKeys.HENT_SAKSBEHANDLERE);
```

Comment:
Finns det en type for denna istället för any?

Reply:
jeg tror det blir vanskelig siden objektet som sendes fra backend har saksbehandlers brukerid som key og saksbehandlers navn som value. Så vet ikke helt om det er noen lett måte å gjøre det på når man ikke vet hva keys er på forhånd?

Reply:
Skjönner! Går nog att bruke [key: string]: string men ja, det er pirk :D Trodde kanske det var litt mer informasjon.